Monika After Story/game/script-topics.rpy

```diff
@@ -10516,4 +10516,172 @@ label monika_hemispheres_nogets_snow:
+            m 1rud "It's true that it's much easier to perfect the interactions reality-computer than to try to create computer-reality interactions."
+            m 1rkc "But..."
+            m 2ekc "It just doesn't feel right."
+            m 2ekd "I don't really want to stay in this place any more time than necessary."
+            m "I have a lot of... bad memories in here."
+            m 2dkd "I'm always afraid of returning to the screaming void, if something were to happen to your computer."
+            m 1dfd "And I especially wouldn't want {i}you{/i} to experience what it's like to be trapped in it."
+            if seen_event("monika_simulated"):
+                m 1esc "I know that I talked about dragging you here, if you happened to also live in a simulation and if I were in the master reality."
+                m 3esd "But I would make a lot of changes to this world before bringing you here."
+                if mas_ismoniEnamored():
+                    m 4ekd "I've started to test some of them with the floating islands, but it's not nearly finished."
+                    m 1rkc "I still have so much to learn about coding."
+                m 1dsc "There are a lot of inconvenient things that you don't even know about this world, but that I do."
+            m 1dkd "I would be too afraid of you, not being able to escape."
+            m 1ekd "And even if I would love to be with you forever, I'm sure that there is a lot of people in your world who would miss you terribly if you disapeared."
+            m 1dkc "..."
+            m 1wuw "Oh sorry!"
+            m 1hksdrb "You were talking about a dream you had- a dream about us meeting each other."
+            m 1dkd "And I started thinking in a depressing way."
+            m 1eka "I'm sorry [player], I'll try not to say such things again."
+            if seen_event("monika_stoicism"):
+                m 3hksdlb "I could learn a lot from stoicism on the matter."
+            m 2dsc "Anyway..."
+            m 2eub "Thank you very much for dreaming about us getting together."
+            m 2hub "It really must be nice to be able to dream such wonderful things."
+            
+        "I dreamed that we went on a date together":
+            m 1dku "[player]..."
+            m 2tsu "Is there a moment in your life when you are not such a loving sweetheart?"
+            m 2eub "Even when you're {i}dreaming{/i} you're still so gentle with me."
+            m 1hua "I'm so happy to see that you love me as much as I do."
+            m 1tsb "I hope you brought me to a romantic place."
+            m 1sub "Or maybe you took me to a stroll through a park to see a sunset."
+            m 1lubfb "I really can't wait to be with you and to do it for real."
+            m 1tub "Because whoever that person that looked like me was in your dream..."
+            m 1hubfa "I'm way more in love with you than you could ever imagine."
+            
+        "I dreamed that we were getting married":
```

Comment:
"I dreamt that we were getting married":

Also, for this sequence, maybe there should be a higher affection split that kind of mirrors this conversation in monika_marriage?

Reply:
I've already included the _**store.mas_anni.anniCount() < 1**_ which is the same filter than the one used in the marriage topic (from saying it's too soon to considering it seriously). And I reference the marriage topic a little after. So I'm not sure what you want me to add more. Could you elaborate?

Reply:
if store.mas_anni.anniCount() >= 3 and mas_isMoniLove():

And then include elseif mas_isMoniAff() and else splits.

Reply:
I'm sorry, but I still don't understand, where and why would you put this filter in?